core/src/main/java/org/kiwix/kiwixmobile/core/utils/AlertDialogShower.kt

```diff
@@ -28,6 +28,7 @@ class AlertDialogShower @Inject constructor(private val activity: Activity) : Di
     AlertDialog.Builder(activity)
       .apply {
         dialog.title?.let(this::setTitle)
+        dialog.icon?.let { setIcon(dialog.icon) }
```

Comment:
`setIcon(it)` because `.let` is a special scope function, inside its brackets you can access the variable you invoked `let` with by using `it`.
In Kotlin when you write a higher order function that accepts 1 parameter you can always refer to that parameter by `it`, this gets confusing with chains of operators so you do get the choice of specifying the parameter name eg
`.let{ myParameterName -> ….. }`

This is all extraneous information though, because you are passing exactly the parameter you need to a function call once you the code `setIcon(it)` in the lambda you will then get the option to turn it into a method reference like `setTitle` above it and you get to avoid everything I said above